articles/ros_node_idl.md

```diff
@@ -97,59 +104,128 @@ This proposal has a number of potential upsides, but it also has some downsides
 ### This is only really useful if it gains significant adoption in upstream packages
 
 It's true that, if not all of the packages in one's system have adopted this, its gains are incomplete.
+
```

Comment:
You're actually making entirely new paragraphs every time you do this. They should be part of the _same_ paragraph, but ROS conventions are to keep each sentence on its own line.